.env

```diff
@@ -0,0 +1,4 @@
+INFLUXDB_HOST=influxdb
+INFLUXDB_PORT=8086
+INFLUXDB_DATABASE=beehive_data
+PORT=5000
```

Comment:
(optional) for consistency with other services, I'd probably prefer loading config from some js file like I do in: 

https://github.com/Gratheon/user-cycle/blob/main/src/config/index.ts
https://github.com/Gratheon/user-cycle/blob/main/src/config/config.default.ts

But can be done later